data/frame.go

```diff
@@ -56,6 +56,13 @@ func (f *Frame) AppendRow(vals ...interface{}) {
 	}
 }
 
+// InsertRowAt adds values in the field vector (identyfied by fieldIdx) at the index specified by vectorIdx
+func (f *Frame) InsertRowAt(fieldIdx int, vals ...interface{}) {
```

Comment:
- I think this is backwards I think, fieldIdx should be rowIdx.
- Repeat the comments about append and panic for InsertAt.

Based on another conversation, it would be nice to have SetRowAt (Frame only). Would you mind adding that as well since it is kind of strange to have Insert but not Set.